code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/AuthConfigsManager.java

```diff
@@ -0,0 +1,47 @@
+package io.cattle.platform.iaas.api.auth;
+
+import io.cattle.platform.iaas.api.auth.integration.interfaces.AuthConfig;
+import io.cattle.platform.iaas.api.auth.integration.interfaces.AuthConfigManager;
+import io.github.ibuildthecloud.gdapi.factory.SchemaFactory;
+import io.github.ibuildthecloud.gdapi.model.ListOptions;
+import io.github.ibuildthecloud.gdapi.request.resource.impl.AbstractNoOpResourceManager;
+
+import java.util.Map;
+import javax.inject.Inject;
+
+public class AuthConfigsManager extends AbstractNoOpResourceManager {
```

Comment:
That part hasn't had the naming figured out yet. Still trying to get it to work.